### PR TITLE
Add eslint support with empty ruleset

### PR DESCRIPTION
**Pull request by sandersn**
1. Add base config at @definitelytyped/eslint. This config has NO rules turned on yet.
2. dtslint now runs this base config, or .eslintrc.json if present in a DT package.
3. No support for per-version runs or anything yet. Not sure yet if I'll do that in a separate PR or this one.

### CODE REVIEW

**Review by andrewbranch — APPROVED** [review]

**Comment by sandersn** [comment]
1. I'm going to wait until my DT week to merge this so @jakebailey doesn't have to watch for errors it introduces.
2. CI is having trouble with the new package since I merged from master. I'm going to try fiddling with the version numbers since it's not failing locally.

**Pull request merged**
